powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/controller/SignatureController.java

```diff
@@ -68,12 +70,9 @@ public Response validateSignature(PowerAuthApiAuthentication auth) throws PowerA
             logger.debug("Signature validation failed");
             throw new PowerAuthSignatureInvalidException();
         }
-        if (!"3.0".equals(auth.getVersion())
-                && !"3.1".equals(auth.getVersion())
-                && !"3.2".equals(auth.getVersion())) {
-            logger.warn("Endpoint does not support PowerAuth protocol version {}", auth.getVersion());
```

Comment:
@romanstrobl May we pass these context messages to the exception message?

Reply:
What do you mean?

Reply:
I mean to call `new PowerAuthInvalidRequestException("Endpoint does not support PowerAuth protocol version " + auth.getVersion())`

Reply:
@banterCZ Currently, we do not do it (we need to have this in the log).

Reply:
It is not crucial, but my idea was to have it both in the log and in the exception message.